### PR TITLE
Test automatically

### PR DESCRIPTION
**Pull request by potsbo**
## WHY

from https://github.com/wantedly/developers-account-mapper/issues/8
Currently, we test only that we can build or not.
We need more specific test

## WHAT

- [x] Add `test` test to Makefile
- [x] Call it in travis

Merging https://github.com/wantedly/developers-account-mapper/pull/11 makes diff in this PR clean.

### CODE REVIEW

**Comment by potsbo** [comment]
Ready to be reviewed and merged

File: Makefile [hunk 1]
```diff
@@ -25,6 +25,9 @@ ifeq ($(shell command -v glide 2> /dev/null),)
 	curl https://glide.sh/get | sh
 endif
 
+test:
```
Comment (dtan4) [inline Makefile:29]: Please add `.PHONY: test` 🙏

**Review by dtan4 — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (potsbo) [reply Makefile:29]: @dtan4 Added, nice review, thanks

**Review by dtan4 — APPROVED** [review]

**Comment by dtan4** [comment]
LGTM 👍

**Pull request merged**